src/Controls/samples/Controls.Sample/XamlApp.xaml

```diff
@@ -2,10 +2,8 @@
     xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
     xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
     xmlns:local="clr-namespace:Maui.Controls.Sample"
-    xmlns:windows="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific;assembly=Microsoft.Maui.Controls"
     xmlns:converters="clr-namespace:Controls.Sample.Converters"
-    x:Class="Maui.Controls.Sample.XamlApp"
-    windows:Application.ImageDirectory="Assets">
+    x:Class="Maui.Controls.Sample.XamlApp">
```

Comment:
The `windows:Application.ImageDirectory="Assets"` is no longer needed by default.